tests/everest/test_everest_initialization.py

```diff
@@ -5,13 +5,10 @@
 from ert.run_models.everest_run_model import EverestRunModel
 from everest.config import EverestConfig
 
-NO_PROJECT_RES = (
-    os.environ.get("NO_PROJECT_RES", False),
-    "Skipping tests when no access to /project/res",
-)
+from .utils import skipif_no_everest_models
 
 
-@pytest.mark.skipif(NO_PROJECT_RES[0], reason=NO_PROJECT_RES[1])
+@skipif_no_everest_models
```

Comment:
This test used to be skipped if eclipse was unvailable but now it will be skipped if `everest-models` is not installed. Is this functional change intentional?

Reply:
Right, that is an oversight, intended to add:
```
@pytest.mark.requires_eclipse
```
though I think both are required. It was just that the no_project_res marker was misleading.

Will fix it